internal/telemetry/tracker.go

```diff
@@ -96,6 +96,7 @@ func (t *tracker) defaultCommandOptions() []EventOpt {
 		withService(config.Default()),
 		withProjectID(t.cmd, config.Default()),
 		withOrgID(t.cmd, config.Default()),
+		withOutput(t.cmd, config.Default()),
```

Comment:
Do we need to update the logic in the monolith too? I think the logic there sends all the properties without checking the prop name but it may be worth double-checking

Reply:
yes, this is just a map of values for telemetry so we can always add to it